auth/register.go

```diff
@@ -1 +1,63 @@
-package auth
\ No newline at end of file
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var client *mongo.Client
```

Comment:
A client is already created which includes methods to perform basic queries in the utils folder. Check utils/db.go. You can easily import and use the functions